src/services/BiometricAuth.js

```diff
@@ -27,44 +28,50 @@ export default class BiometricAuth extends React.PureComponent {
 		super(props);
 		this.state = {
 			showModal: false,
-			accepted: false,
 		};
 	}
 
-	async authenticate(successCallback, errorCallback, message = t('default_message')) {
-		const { restrictions, action } = this.props;
+	async authenticate(successCallback, errorCallback) {
+		const { restrictions, action, navigation, dispatch } = this.props;
```

Comment:
But now, you cannot set the message yourself ? It shows only the default one ?

Reply:
For now there's only one message